Source/Core/AudioCommon/OpenALStream.cpp

```diff
@@ -314,23 +324,46 @@ void OpenALStream::SoundLoop()
     // many silence samples.  These do not need to be timestretched.
     if (rate > 0.10)
     {
-      soundTouch.setTempo(rate);
-      if (rate > 10)
+      if (use_timestretching)
+      {
+        soundTouch.setTempo(rate);
+      }
+      else
+      {
+        alSourcef(ui_source, AL_PITCH, static_cast<ALfloat>(rate));
```

Comment:
Yes, it sounds normal on a Mac (though the optimal latency depends on what game is running / what else my computer is doing). I imagine this is probably because OpenAL doesn't do timer-based scheduling like the other backends, and, now that I look at it, XAudio2 doesn't seem to either...